saddle-core/src/main/scala/org/saddle/Frame.scala

```diff
@@ -1995,6 +2041,22 @@ object Frame extends BinOpFrame {
     }
   }
 
+  /** Factory method to create a Frame from a sequence of tuples, where the
+    * first element of the tuple is a column label, and the second a series of
+    * values. The row labels of the result are the outer join of the indexes of
+    * the series provided. Throws IllegalArgumentException if the row indices
+    * are not unique.
+    */
+  def fromColumns[RX: ST: ORD, CX: ST: ORD, T: ST](
```

Comment:
@pityka  you didn't agree with this one? 
If you list the method names of `Frame`, I think you won't find a single occurrence of `column` in full. But tens of occurences of `col` abbreviated: `sortedCols`, `firstCol`, etc. I think it's more coherent to stick to that naming scheme.

Reply:
Ah yes. Sorry for go ahead without discussion. 

- Indeed I dont like the abbreviations so at least for new values I use proper names. I favor local readability over consistency here. 

- I am not sure on the name of this very method actually because I wanted something to reflect the difference in behavior. However I failed to come up with one so far. 

That said I did not yet cut a release ouf of it so it is still up to possible change.

Reply:
I think `col` in the context of this library is unambiguous.  Also brevity can be a readability gain as well, I'd argue :). 

But regardless of the readability argument, I find consistent naming patterns in a library very valuable. Users rely on them to discover new parts of the api. I am in favor of sticking to consistent naming patterns, even when they arguably aren't the best ones. There can be a separate track on whether the patterns should be changed across the whole library.